pylinac/winston_lutz.py

```diff
@@ -179,13 +196,20 @@ def max_distance_to_lines(p, lines) -> float:
         result = optimize.minimize(max_distance_to_lines, initial_guess, args=things, bounds=bounds)
         return result
 
+    @property
+    def json(self):
+        """Produces a dictionary object with the summary of analysis which is jsonable, should be part of API/ABI"""
+        return self._json
```

Comment:
This is where all the logic should occur; i.e. creating the dictionary keys and values. This way the logic is self-contained and any additions or changes are localized here instead of spread throughout the module as below. E.g. for `gantry_iso_size` the key will never be entered if there are no gantry images, rather than adding a key of 0. If a user always expected this key to be there (a reasonable assumption) a call for this key would fail ungracefully in this case unless they were really careful.

Reply:
I know that's standard practice, but, what I learned from the CBCT module is that there's a huge risk with maintenance. I understand this could be mitigated with unit testing. My thinking was going forward any refactoring would be easier to manage with the current design. 

I will switch to the unit testing ideology and resubmit